src/main/java/com/webperside/courseerpbackend/services/student/StudentBusinessServiceImpl.java

```diff
@@ -48,4 +51,14 @@ public void addStudent(StudentPayload studentPayload) {
         studentService.insert(Student.builder().userId(user.getId()).build());
 
     }
+
+    @Override
+    public void addStudentToGroup(long studentId, long groupId) {
+        if(studentService.checkByStudentId(studentId)){
```

Comment:
it would be great to name it more specifically for example, checkStudentAlreadyAddedToGroup

Reply:
before that checking, you have to check the existence of the student and the group. Currently, you can use findByid method (if there is no method like that, please create it) (as you know if there is not data with this ID, this method has to throw not found exception. you can look other services for instance). 
We'll create a more reusable method for checking the existence of any data. (it will be a new task, currently, you can use the method I mentioned above